tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -4723,9 +4731,7 @@ <h2>Show similarity to selected datapoint</h2>
         });
         this._requestManager.request(url, postData).then(wrapperFn)
             .catch(reason => {
-              this.exampleStatusStr = 'Request failed';
-              this.showToast_('Request failed: ' + reason);
-              this.$.spinner.hidden = true;
+              this.handleError_('Request failed: ' + reason);
```

Comment:
good point. Adding new string param for what the request does, so that if it fails the error states what request failed (such as 'Datapoint load', or 'Model inference').